tests/e2e/tests/solution-server/settings.test.ts

```diff
@@ -0,0 +1,119 @@
+import { RepoData, expect, test } from '../../fixtures/test-repo-fixture';
+import { VSCode } from '../../pages/vscode.page';
+import { KAIViews } from '../../enums/views.enum';
+import { extensionName } from '../../utilities/utils';
+import { OPENAI_GPT4O_PROVIDER } from '../../fixtures/provider-configs.fixture';
+
+type SolutionServerConfig = {
+  name: string;
+  ssEnabled: boolean;
+  authInIDE: boolean;
+  insecure: boolean;
+  realm: string;
+  shouldConnect: boolean;
+  username?: string;
+  password?: string;
+};
+
+const solutionServerConfigs: SolutionServerConfig[] = [
+  {
+    name: 'All enabled',
+    ssEnabled: true,
+    authInIDE: true,
+    insecure: true,
+    realm: 'mta',
```

Comment:
`realm`, `user` and `password` should be provided by env variables. I think @RanWurmbrand  was working on enabling solution server configuration with env variables so he may update this later.


Also `realm` should default to `konveyor`.

Reply:
Yes, right!